extensions/amp-story/amp-story.md

```diff
@@ -596,6 +600,10 @@ Every element inside an `<amp-story-page>` can have an entrance animation.
 
 You can configure animations by specifying a set of [animation attributes](#animation-attributes) on the element; no additional AMP extensions or configuration is needed.
 
+{% call callout('Note', type='note') %}
+Animations can help make your Web Story more visually exciting and engaging, but use them sparingly. Some users may find long, continuous animations distracting. Other users may have motion sensitivity and be adversely affected by excessive use of motion and parallax effects.
```

Comment:
We should also note that animations will be displayed in their end state (as if they were finished playing) for users who disabled animations (reduce motion iOS setting, or remove animation Android setting).